src/Type/Input/DateInput.php

```diff
@@ -1,28 +0,0 @@
-<?php
-namespace WPGraphQL\Type\Input;
-
-class DateInput {
-	public static function register_type() {
```

Comment:
Since this is probably a pretty widely used input arg, I'd rather see us deprecate the existing field and introduce the new field separately. We could then make the breaking change in a future release.